src/views/product-downloads-view/server.ts

```diff
@@ -0,0 +1,77 @@
+import { GetStaticPropsResult } from 'next'
+import { ProductData } from 'types/products'
+import { generateStaticProps as generateReleaseStaticProps } from 'lib/fetch-release-data'
+import { getInlineContentMaps } from 'lib/tutorials/get-inline-content-maps'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import { formatCollectionCard } from 'components/collection-card/helpers'
+import { formatTutorialCard } from 'components/tutorial-card/helpers'
+import {
+  FeaturedTutorialContent,
+  ProductDownloadsViewStaticProps,
+  RawProductDownloadsViewContent,
+  FeaturedTutorialCard,
+} from './types'
+
+/**
+ * Prepares static props for downloads views
+ */
+async function generateStaticProps(
+  productData: ProductData,
+  pageContent: RawProductDownloadsViewContent
+): Promise<GetStaticPropsResult<ProductDownloadsViewStaticProps>> {
+  /**
+   * Fetch the release data static props
+   */
+  const { props: releaseProps, revalidate } = await generateReleaseStaticProps(
+    productData
+  )
+  const { releases, product, latestVersion } = releaseProps
+
+  /**
+   * Fetch page content
+   * (loaded from .json, then tutorial data is fetched)
+   */
+  const {
+    doesNotHavePackageManagers,
+    featuredTutorials,
+    packageManagerOverrides,
+    sidecarMarketingCard,
+  } = pageContent
+  // Gather tutorials and collections based on slugs used
+  const inlineContent = await getInlineContentMaps(featuredTutorials)
+  // Transform feature tutorial and collection entries into card data
+  const featuredTutorialCards: FeaturedTutorialCard[] = featuredTutorials.map(
+    (entry: FeaturedTutorialContent) => {
+      const { collectionSlug, tutorialSlug } = entry
+      if (typeof collectionSlug == 'string') {
+        const collectionData = inlineContent.inlineCollections[collectionSlug]
+        return { type: 'collection', ...formatCollectionCard(collectionData) }
+      } else if (typeof tutorialSlug == 'string') {
+        const tutorialData = inlineContent.inlineTutorials[tutorialSlug]
+        const defaultContext = tutorialData.collectionCtx.default
+        const tutorialLiteCompat = { ...tutorialData, defaultContext }
+        return { type: 'tutorial', ...formatTutorialCard(tutorialLiteCompat) }
+      }
+    }
+  )
```

Comment:
great question - I'm not sure the original term was finessed all that much, it's a holdover from [the learn repo](https://github.com/hashicorp/learn/blob/79fa1971f60b30038d2ee8da0ed056497994c900/pages/consul/index.jsx#L10). I think the intent is "stuff that authors reference in their content, without having to type out the full data". 

I'm definitely open to renaming, but would prefer to handle outside this PR - there's a bunch of places we'd need to change it! Would personally lean towards deferring the rename till dev-dot is in GA (since related utilities in `hashicorp/learn` still have to be maintained)

Reply:
This makes sense! Totally reasonable to wait to rename until GA when there's less to maintain. 💖